<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="scan">
      <attribute name="title">
       <string>Scan</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QCheckBox" name="jpeg">
           <property name="text">
            <string>Use scanner JPEG compression</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QCheckBox" name="limit">
             <property name="maximumSize">
              <size>
               <width>170</width>
               <height>32767</height>
              </size>
             </property>
             <property name="text">
              <string>Limit pages to scan</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="single"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QCheckBox" name="stackLimit">
             <property name="maximumSize">
              <size>
               <width>170</width>
               <height>32767</height>
              </size>
             </property>
             <property name="text">
              <string>Limit pages per stack</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="stackCount"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox1">
           <property name="title">
            <string>Automatic Duplexing</string>
           </property>
           <layout class="QHBoxLayout">
            <item>
             <layout class="QVBoxLayout">
              <item>
               <layout class="QHBoxLayout">
                <item>
                 <widget class="QLabel" name="textLabel3">
                  <property name="text">
                   <string>Blank pages</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="blank">
                  <item>
                   <property name="text">
                    <string>Record</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ignore if back page</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Ignore</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="SliderSpin" name="threshold" native="true">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" stretch="0,2,1">
                <item>
                 <spacer>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="coverageStr_2">
                  <property name="text">
                   <string>Coverage</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="coverageStr">
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>220</red>
                        <green>220</green>
                        <blue>220</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Light">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Midlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>242</red>
                        <green>242</green>
                        <blue>242</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Dark">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>78</red>
                        <green>78</green>
                        <blue>78</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Mid">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>183</red>
                        <green>183</green>
                        <blue>183</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="BrightText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="ButtonText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>220</red>
                        <green>220</green>
                        <blue>220</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Shadow">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>84</red>
                        <green>112</green>
                        <blue>152</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="HighlightedText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Link">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="LinkVisited">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>220</red>
                        <green>220</green>
                        <blue>220</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Light">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Midlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>242</red>
                        <green>242</green>
                        <blue>242</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Dark">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>78</red>
                        <green>78</green>
                        <blue>78</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Mid">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>183</red>
                        <green>183</green>
                        <blue>183</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="BrightText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="ButtonText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>220</red>
                        <green>220</green>
                        <blue>220</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Shadow">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>84</red>
                        <green>112</green>
                        <blue>152</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="HighlightedText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Link">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="LinkVisited">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>128</red>
                        <green>128</green>
                        <blue>128</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>220</red>
                        <green>220</green>
                        <blue>220</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Light">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Midlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>242</red>
                        <green>242</green>
                        <blue>242</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Dark">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>78</red>
                        <green>78</green>
                        <blue>78</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Mid">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>183</red>
                        <green>183</green>
                        <blue>183</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>183</red>
                        <green>183</green>
                        <blue>183</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="BrightText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="ButtonText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>128</red>
                        <green>128</green>
                        <blue>128</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>220</red>
                        <green>220</green>
                        <blue>220</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Shadow">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>0</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>128</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="HighlightedText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Link">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="LinkVisited">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>0</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="text">
                   <string>3.4%</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="files">
      <attribute name="title">
       <string>Files</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Public group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="group">
           <property name="toolTip">
            <string>Group name to use for files. This can make it easier to share
files with others. Leave blank for the default group</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>199</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SliderSpin</class>
   <extends>QWidget</extends>
   <header>sliderspin.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">sliderspin.h</include>
  <include location="local">sliderspin.h</include>
  <include location="local">sliderspin.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>ok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>cancel_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok_clicked()</slot>
  <slot>cancel_clicked()</slot>
 </slots>
</ui>
